### ISSUE TITLE
add option to restart timeline when selecting new ghost

### ISSUE THREAD

**Issue by ezio416** [XertroV/tm-ghosts-plus-plus#14]
In vanilla, when I select a new ghost to watch, it watches it from the beginning. With G++, the timer does not reset, so selecting a new ghost resumes from the same time that the previous ghost was. I would like a setting for the vanilla behavior

**Comment by XertroV** [comment]
yep, sure. It's something I added b/c I don't like that you can't swap ghosts and keep the old ghost loaded. 
But I can do both ofc by adding a setting which sets timer to 0 on load.
